pytac/lattice.py

```diff
@@ -349,6 +326,65 @@ def set_element_values(self, family, field, values):
         if len(elements) != len(values):
             raise LatticeException("Number of elements in given array must be"
                                    " equal to the number of elements in the "
-                                   "family")
+                                   "family.")
         for element, value in zip(elements, values):
-            element.set_value(field, value)
+            element.set_value(field, value, handle=pytac.SP)
+
+    def set_default_arguments(self, default_units=None,
+                              default_data_source=None):
+        """Sets the default handle, units and data_source values for the lattice
+        and all its elements.
+
+        Args:
+            default_handle (str): The default handle to be set across the entire
+                                   lattice, pytac.RB or pytac.SP.
+            default_units (str): The default unit type to be set across the
+                                  entire lattice, pytac.ENG or pytac.PHYS.
+            default_data_source (str): The default data source to be set across
+                                        the entire lattice, pytac.LIVE or
+                                        pytac.SIM.
+
+        Raises:
+            LatticeException: if no default arguments are given.
+            UnitsException: if specified default unit type is not a valid unit
+                             type.
+            DeviceException: if specified default data source is not a valid
+                              data source.
+        """
+        if not any([default_units, default_data_source]):
+            raise LatticeException('Please set at least one default argument '
+                                   'for units or data_source.')
+        if default_units == pytac.ENG or default_units == pytac.PHYS:
+            self._data_source_manager._default_units = default_units
```

Comment:
You're using a private attribute `_default_units` from outside the DataSourceManager class. Should it be a public attribute?